JapaneseFilter/sections/specific.txt

```diff
@@ -651,6 +651,7 @@ my-best.com##.l-footer__fixed
 print-kids.net##div[id^="ad-"]
 print-kids.net##div[id$="-rectangle"]
 twidropper.appspot.com,print-kids.net##.adsense-responsive
+cojicaji.jp##.adsense-below-article
```

Comment:
Change to `##div[class^="adsense-below"]` and add `##.dfp-cci-rectangle-wrapper` for mobile.

Reply:
ok , fixed